### PR TITLE
Fix dashboard phone widget styling

### LINKED ISSUE #1577: Improve phone number prefix field rendering
The country prefix and phone number fields render in two lines:
image
Instead, we should render those fields in one line, similarly to how it looks like in the storefront:
image
I think there is no such pattern in Material Design and we could simply put the two fields next to each other, preserving the with-proportions like in the storefront.

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Resolves #1577

### CODE REVIEW

File: saleor/dashboard/widgets.py [hunk 1]
```diff
@@ -15,3 +23,17 @@ def __init__(self, attrs=None):
         self.currency = getattr(settings, 'DEFAULT_CURRENCY')
         widgets = (PriceInput(self.currency), PriceInput(self.currency))
         super(RangeWidget, self).__init__(widgets, attrs)
+
+
+class PhonePrefixWidget(PhoneNumberPrefixWidget):
```
Comment (elwoodxblues) [inline saleor/dashboard/widgets.py]: We only need to customize the `template_name` property, so this widget should override the `saleor.userprofile.widgets.PhonePrefixWidget` class.

File: saleor/dashboard/order/forms.py [hunk 2]
```diff
@@ -356,3 +358,8 @@ def save(self):
         add_variant_to_delivery_group(
             self.group, variant, quantity, self.discounts)
         recalculate_order(self.group.order)
+
+
+class DashboardAddressForm(AddressForm):
```
Comment (elwoodxblues) [inline saleor/dashboard/order/forms.py]: We already know that this is a "dashboard" form by the path of this module, so there's no need to mention that in the name. If you're trying to solve the problem of conflicting names, you should rather change the name of the imported `AddressForm`:
```from ...userprofile.i18n import AddressForm as StorefrontAddressForm```

Comment (dominik-zeglen) [reply saleor/dashboard/order/forms.py]: Damn, I completely forgot about `import ... as ...`, thank you

File: saleor/dashboard/order/views.py [hunk 3]
```diff
@@ -17,15 +17,14 @@
     AddVariantToDeliveryGroupForm, CancelGroupForm, CancelLinesForm,
     CancelOrderForm, CapturePaymentForm, ChangeStockForm, ChangeQuantityForm,
     MoveLinesForm, OrderNoteForm, RefundPaymentForm, ReleasePaymentForm,
-    RemoveVoucherForm, ShipGroupForm)
+    RemoveVoucherForm, ShipGroupForm, DashboardAddressForm)
```
Comment (rafalp) [inline saleor/dashboard/order/views.py]: Other imports here are ordered alphabetically, so shouldn't `DashboardAddressForm ` go in between `ChangeQuantityForm` and `MoveLinesForm`?

File: saleor/dashboard/widgets.py [hunk 4]
```diff
@@ -15,3 +18,7 @@ def __init__(self, attrs=None):
         self.currency = getattr(settings, 'DEFAULT_CURRENCY')
         widgets = (PriceInput(self.currency), PriceInput(self.currency))
         super(RangeWidget, self).__init__(widgets, attrs)
+
+
+class PhonePrefixWidget(StorefrontPhonePrefixWidget):
+    template_name = 'dashboard/order/widget/phone-prefix-widget.html'
```
Comment (elwoodxblues) [inline saleor/dashboard/widgets.py]: We have recently settled on a naming convention for the template files to use underscores. Dashes should be used for static files that are accessible from an URL.

Comment (dominik-zeglen) [reply saleor/dashboard/widgets.py]: Right!

**Pull request merged**